fdbserver/storageserver.actor.cpp

```diff
@@ -1906,6 +1919,9 @@ ACTOR Future<Void> getKeyValuesQ(StorageServer* data, GetKeyValuesRequest req)
 {
 	state Span span("SS:getKeyValues"_loc, { req.spanContext });
 	state int64_t resultSize = 0;
+	state IKeyValueStore::ReadType type =
+	    req.isFetchKeys ? IKeyValueStore::ReadType::FETCH : IKeyValueStore::ReadType::NORMAL;
+	getCurrentLineage()->modify(&TransactionLineage::txID) = req.spanContext.first();
```

Comment:
This line seems to be the source of the CI failure.

Reply:
```suggestion
```